core/src/main/java/org/polypheny/db/catalog/snapshot/Snapshot.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2019-2023 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.catalog.snapshot;
+
+import java.util.List;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.jetbrains.annotations.NotNull;
+import org.polypheny.db.adapter.java.AdapterTemplate;
+import org.polypheny.db.algebra.constant.FunctionCategory;
+import org.polypheny.db.algebra.constant.Syntax;
+import org.polypheny.db.algebra.operators.OperatorTable;
+import org.polypheny.db.catalog.entity.LogicalAdapter;
+import org.polypheny.db.catalog.entity.LogicalAdapter.AdapterType;
+import org.polypheny.db.catalog.entity.LogicalQueryInterface;
+import org.polypheny.db.catalog.entity.LogicalUser;
+import org.polypheny.db.catalog.entity.logical.LogicalEntity;
+import org.polypheny.db.catalog.entity.logical.LogicalNamespace;
+import org.polypheny.db.catalog.entity.logical.LogicalTable;
+import org.polypheny.db.catalog.logistic.Pattern;
+import org.polypheny.db.config.RuntimeConfig;
+import org.polypheny.db.iface.QueryInterfaceManager.QueryInterfaceTemplate;
+import org.polypheny.db.nodes.Identifier;
+import org.polypheny.db.nodes.Operator;
+import org.polypheny.db.util.NameMatcher;
+import org.polypheny.db.util.NameMatchers;
+
+public interface Snapshot extends OperatorTable {
+
+    NameMatcher nameMatcher = NameMatchers.withCaseSensitive( RuntimeConfig.RELATIONAL_CASE_SENSITIVE.getBoolean() );
```

Comment:
Case-sensitivity is defined by the namespace. We should get rid of this runtime config. There might only be a config for the default when creating a new namespace.

Reply:
Changed runtime config to default case when creating namespaces.